Add ryanolsonx/sublimetext-zenburn-theme

ryanolsonx opened the pull request:
Ported the zenburn color scheme to sublime text. I have a VS Code version and would love to provide this in package control for sublime text users. This is vastly different than the existing zenburn theme. My theme is a lot closer to the Vim color scheme (that this was ported from).

<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks,
depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request
and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass.
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can request a review from @packagecontrol-bot
to manually trigger an automated review
if you don't need to push a new commit.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"`
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.

You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists,
why you believe it is different and needed
below this line. -->

braver commented:
The existing package is actively maintained, why not open a PR there?

ryanolsonx commented:
Okay - I'll work with that package author to see if I can incorporate my changes into that repo. If not, would we be able to get this in as a separate package, @braver?

braver commented:
Sure, there is nothing fundamentally against multiple versions of the same idea, but all around it’s better if we can work together and present users with a single really good version.

braver commented:
I’ll close this for now, just ping me to let me know if you worked it out or if we need to look at adding your version as well.

ryanolsonx commented:
@braver  So far no response from @colinta after 3 days. I created an issue https://github.com/colinta/zenburn/issues/8 and a PR https://github.com/colinta/zenburn/pull/9. If there's no response by Jul 27th (after 1 week), would you mind re-opening this?

braver commented:
We usually keep a 2 week timeout period for these things. @colinta seems to regularly check in in GitHub, so let's give him a little time.

rwols commented:
Three days is very short for people doing this in their spare free time ;)

ryanolsonx commented:
@rwols I was thinking 1 week would be reasonable -- not 3 days.

I'm happy to wait 2 weeks 😃

ryanolsonx commented:
@braver Do you mind re-opening this?

braver commented:
Alright, let's do this. 

- You have quite a few files that aren't functional to the package. You can exclude them from the package using [.gitattributes and export-ingore](https://github.com/SublimeLinter/SublimeLinter/blob/master/.gitattributes), saving some storage bits and CO2 ;) 
- About the name, I think it should signal this is the new and improved Zenburn. "Zenburn New", "Zenburn Improved", something like that?
- And have a look at the feedback from the bot.

ryanolsonx commented:
@braver  Alright - so I've fixed up a lot of the issues. 

- I removed the "demo" directory that wasn't really needed.
- It has a license file now.

Let's talk about the name. I don't feel like the existing Zenburn color theme is really "Zenburn." Mine is a more accurate port of what Zenburn is all about and really matches up with what was originally created for Vim. With that said, would we be able to keep my proposed name? I feel like it would be misleading to users for them to see Zenburn Improved or something like that when I haven't changed the nature of the Zenburn theme itself (and as it's not a variant of Zenburn).

Thoughts?

braver commented:
Well you still have a lot of screenshots in the package, I would encourage you to use the `.gitattributes` for that. I also kinda doubt that they're useful to new users.

I agree with your reasoning about the name though, let's keep it as it is.

ryanolsonx commented:
Awesome.

I'll remove those screenshots. It should be much better now.

ryanolsonx commented:
Thanks a lot @braver!
